src/client.ts

```diff
@@ -4130,7 +4130,7 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
     public setPowerLevel(
         roomId: string,
         userId: string | string[],
-        powerLevel: number,
+        powerLevel: number | undefined,
```

Comment:
Wondering whether the following would be more adequate? Or whether it's better to be really explicit?

```suggestion
        powerLevel?: number,
```

Reply:
Can't because the next field isn't optional, optional fields cannot be followed by mandatory fields

Reply:
Woops, i read that wrong and thought we destructured the thing